Tests/GarageStorageTests/RawStorageTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RawStorageTests.swift
+//  
+//
+//  Created by Brian Arnold on 6/7/24.
+//
+
+import XCTest
+@testable import GarageStorage
+
+class TestCoreData: Codable { }
+
+final class RawStorageTests: XCTestCase {
+
+    override func setUpWithError() throws {
```

Comment:
#q: Is this set up function needed if we create a new `Garage` instance on line 22?

Reply:
@kyleronayne Yes it's needed, but it's not obvious why it's needed, so I'm glad you brought this up.

Basically, because none of the test code initializing a Garage specifies a persistent store description, they all wind up sharing the same default persistent store description. This means they all save to the same local storage file, which must be reset before each unit test is run.

I created https://github.com/mrlegowatch/GarageStorageSwift/issues/23 to track updating the test code to change this for all of the unit test code, such that any garage's that wish to share a persistent store description, must at least indicate the same store name, and to remove the default persistent store description from the implementation.